tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -171,6 +171,11 @@ test_i2c_ms5837: ms5837@1a {
 	reg = <0x1a>;
 };
 
+test_i2c_mcp9600: mcp9600@60 {
+	compatible = "microchip,mcp9600";
+	reg = <0x60>;
+};
+
```

Comment:
Have you rebased to the latest main recently? These addresses should be in numerical order

Reply:
Appended in correct numerical order